Dockerfile

```diff
@@ -14,15 +14,33 @@
 # limitations under the License.
 #
 
-# This image is for the FHIR proxy with configuration knobs as environment vars.
-
-FROM adoptopenjdk/maven-openjdk11
+# Image for building and running tests against the source code of
+# the FHIR Access Proxy.
+FROM maven:3.8.5-openjdk-11 as build
 
 RUN apt-get update && apt-get install -y nodejs npm
 RUN npm cache clean -f && npm install -g n && n stable
 
 WORKDIR /app
 
+COPY server/src ./server/src
+COPY server/pom.xml ./server/
+COPY plugins/src ./plugins/src
+COPY plugins/pom.xml ./plugins/
+COPY license-header.txt .
+COPY pom.xml .
+
+RUN mvn spotless:check
+RUN mvn --batch-mode package -Pstandalone-app
+
+
+# Image for FHIR Access Proxy binary with configuration knobs as environment vars.
+FROM eclipse-temurin:11-jre-alpine as main
+RUN apk --no-cache add curl
```

Comment:
So as discussed offline, please take a look at the base image options and pick something that is not too hard to debug. The current option (i.e., `jre-alpine`) is fine if it has something like bash. If not, we can obviously add things that we need (like `curl` here) but maybe there is a better tool-set/size trade-off.

Reply:
switched to an image that is Debian-based: `eclipse-temurin:11-jdk-focal` (contains `bash`, and `curl`)